src/Effects.js

```diff
@@ -479,6 +537,36 @@ module.exports = class Effects {
       }
     };
 
+    this.lasers = {
+      laser: [],
+      draw: function () {
+        p5.background('black');
+        let laser = {
+          w: 200,
+          x: 200,
+          y: 1700,
+          z: 400,
+          color: randomColor(255,255,255),
+        };
+        this.laser.push(laser);
```

Comment:
:-1: Memory leak. We're pushing a new object into `this.laser` every frame and never removing any of them.  Especially considering we don't reset background states across run/reset this could crash the page across enough runs.

Reply:
@islemaster cleaning these up as part of later merge commit. we are going to set the limit to 100 lasers at any given time.